src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/cleanUpData.hpp

```diff
@@ -42,7 +42,7 @@ class TCleanUpData final : public AbstractHandler<std::shared_ptr<TScanContext>>
      * @param inventoryDatabase Inventory database.
      */
     explicit TCleanUpData(std::shared_ptr<TIndexerConnector> indexerConnector, Utils::RocksDBWrapper& inventoryDatabase)
-        : m_indexerConnector(std::move(indexerConnector))
+        : m_indexerConnector(indexerConnector)
```

Comment:
We must confirm that this is the best approach. We can see this in some other classes, so we should know why we were moving the smart pointer in the first place.